modules/installation-azure-create-ingress-dns-records.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_azure/installing-azure-user-infra.adoc
+
+[id="installation-azure-create-ingress-dns-records_{context}"]
+= Adding the Ingress DNS records
+
+If you removed the DNS Zone configuration when creating Kubernetes manifests and
+generating Ignition configs, you must manually create DNS records that point at
+the Ingress load balancer. You can create either a wildcard
+`*.apps.{baseDomain}.` or specific records. You can use A, CNAME, and other
+records per your requirements.
+
+.Prerequisites
+
+* You deployed an {product-title} cluster on Microsoft Azure by using infrastructure that you provisioned.
+* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`.
+* Install the `jq` package.
+* Install or update the link:https://docs.microsoft.com/en-us/cli/azure/install-azure-cli-yum?view=azure-cli-latest[Azure CLI].
```

Comment:
Right now, we can't guarantee that we tested the procedure with the latest `az` cli.
I've tested it with the version highlighted in the github docs and right now latest (2.2.0) from the RPM installation procedure linked here. 
However, I'm already getting deprecation warnings and we could see a different behavior than our example suggests.

Deprecation warning example:

>`Deprecation warnings:
>
>$ az group deployment create -g ${RESOURCE_GROUP} \
>   --template-file "01_vnet.json" \
>   --parameters baseName="${INFRA_ID}"
This command is implicitly deprecated because command group 'group deployment' is deprecated and will be removed in a future release. Use 'deployment group' instead.`

Reply:
I can add a note indicating the CLI that the procedure was tested for.

I noticed the `az` CLI was just updated to 2.2.0 this past Tuesday (Mar 10) with that particular deprecation (release notes [here](https://docs.microsoft.com/en-us/cli/azure/release-notes-azure-cli?view=azure-cli-latest#arm)). I can change all `az group deployment` &rarr; `az deployment group` if that was the only issue during your testing, and indicate the 2.2.0 version in the note. Does that sound good?

Reply:
Updated `az group deployment` &rarr; `az deployment group` to follow latest CLI version (2.2.0). I'll add the note at the beginning of the document.